fastlane/jetpack_metadata/es-ES/release_notes.txt

```diff
@@ -1,5 +1,5 @@
 - Hemos facilitado la visualización de los nuevos bloques disponibles para que los uses en tu sitio.
-- Ahora puedes destacar el texto con la etiqueta HTML <mark>, así como cambiar el texto y el color de fondo del botón del bloque de búsqueda. ¡Perfecto!
+- Ahora puedes destacar el texto con la etiqueta HTML "mark", así como cambiar el texto y el color de fondo del botón del bloque de búsqueda. ¡Perfecto!
```

Comment:
Reminder: All these `<` to `"` changes were done manually because ASC rejects HTML tags in the copy. They went into 18.6.1 because I realized the problem only upon submission, which happened after CI had already tagged. It didn't feel like the kind of change that granted scrapping the tag and pushing a new one, mainly because it's not user facing from the point of view of the app binary.